front/components/workspace/Analytics.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Button,
+  CloudArrowDownIcon,
+  DropdownMenu,
+  Spinner,
+} from "@dust-tt/sparkle";
+import type { WorkspaceType } from "@dust-tt/types";
+
+import { useWorkspaceAnalytics } from "@app/lib/swr";
+
+interface QuickInsightsProps {
+  owner: WorkspaceType;
+}
+
+export function QuickInsights({ owner }: QuickInsightsProps) {
+  const { analytics, isMemberCountLoading } = useWorkspaceAnalytics({
+    workspaceId: owner.sId,
+    disabled: false,
+  });
+
+  if (!analytics) {
+    return null;
+  }
+
+  return (
+    <>
+      <div>
+        <span className="text-md font-semibold">Quick insights</span>
+        {isMemberCountLoading ? (
+          <Spinner />
+        ) : (
+          <div className="mt-2 grid grid-cols-2 gap-2">
```

Comment:
We don't have anythin for these in sparkle?

Reply:
Hmmm should be feasible with `Page.Vertical` and `Page.Horizontal`

Reply:
I meant more generally for all these raw tailwind stuff in that overall component :)